src/events/eventActions.js

```diff
@@ -178,6 +179,12 @@ export const registerAndStartPolling =
             globalSettings: getGlobalSettings(),
           },
         );
+        logging.error(e, {
```

Comment:
N.B. (here and at the other `logging.error` call): the error has the version in its message:

```js
export class ServerTooOldError extends ExtendableError {
  version: ZulipVersion;
  constructor(version: ZulipVersion) {
    super(`Unsupported Zulip Server version: ${version.raw()}`);
    this.version = version;
  }
}
```

So it might unhelpfully split these logs up into separate issues in Sentry.

Reply:
Yeah, I suspect that at least when the version has a Git commit ID in it, it'll get split up into a bunch of different issues for different commit IDs.

Let's put `version.coarse()` in the error message, and include the finer details just in the extras object below.